.github/workflows/test-comment-reaction.yml

```diff
@@ -4,7 +4,9 @@ name: test-comment-reaction
 on:
   issue_comment:
     types:
-      - created
+      # disable for the time being...
+      #- created
+      - deleted
```

Comment:
Curious: if it's meant to be disabled, why add the `deleted` event?

Reply:
We could also do: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow